src/app/plugins/status.plugin.ts

```diff
@@ -0,0 +1,123 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+import { _getPluginStoreSize } from '../../bootstrap/plugin.loader';
+import { RichEmbed } from 'discord.js';
+
+export class StatusPlugin extends Plugin {
+  public name: string = 'Status';
+  public description: string = 'Gets info about Lion';
+  public usage: string = 'status';
+  public pluginAlias = [];
+  public permission: ChannelType = ChannelType.Bot;
+
+  private LION_PFP_LIN: string =
+    'https://cdn.discordapp.com/avatars/574623716638720000/7d404c72a6fccb4a3bc610490f8d7b72.png';
+  private REPO_LINK = 'https://github.com/joey-colon/lion/commit/';
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public validate(message: IMessage, args: string[]) {
+    return args && args.length === 0;
+  }
+
+  public async execute(message: IMessage, args: string[]) {
+    const latestCommit = await Promise.resolve(this._getLatestCommit());
+    const numPluigins = _getPluginStoreSize();
+    const uptime = this._getUptime();
+
+    const embed = this._creatEmbed(latestCommit, numPluigins, uptime);
+    message.reply(embed);
+  }
+
+  private _creatEmbed(latestCommit: any, numPluigins: number, uptime: number) {
+    const commitLink = this.REPO_LINK + latestCommit?.number;
+
+    const embed = new RichEmbed();
+    embed.setTitle('Lion Status');
+    embed.setColor('#1fe609');
+    embed.setThumbnail(this.LION_PFP_LIN);
+    embed.setURL(commitLink);
+
+    embed.addField('Latest Commit Hash', latestCommit?.number, true);
+    embed.addField('Latest Commit Author', latestCommit?.author, true);
+    embed.addField('Latest Commit Date', latestCommit?.date, true);
+    embed.addField('Number Of Plugins', numPluigins, true);
+    embed.addField('Uptime', this._msToTime(uptime), true);
+
+    return embed;
+  }
+
+  //https://stackoverflow.com/questions/19700283/how-to-convert-time-milliseconds-to-hours-min-sec-format-in-javascript
+  private _msToTime(duration: number) {
```

Comment:
you might be able to accomplish this by constructing a `Date` object and `getHours` etc